lib/app/dispatcher.rb

```diff
@@ -60,7 +60,7 @@ def scheduler
       end
 
       def start_polling_jobs!
-        scheduler.when_triggered do |connector_settings|
+        scheduler.when_polling_jobs do |connector_settings, do_sync|
```

Comment:
I'd probably rename `when_polling_jobs` to `with_each_polling_interval_tick` or something similar that indicates how often/when this is triggered

Reply:
`do_sync` by the signature looks like a method in inheritance chain, I'm trying to think of a better name here - does `sync_triggered` sound better?